scripts/skills/actives/rf_gouge_skill.nut

```diff
@@ -0,0 +1,114 @@
+this.rf_gouge_skill <- ::inherit("scripts/skills/skill", {
+	m = {
+		BleedStacks = 3,
+		SoundsA = [
+			"sounds/combat/cleave_hit_hitpoints_01.wav",
+			"sounds/combat/cleave_hit_hitpoints_02.wav",
+			"sounds/combat/cleave_hit_hitpoints_03.wav"
+		]
+	},
+	function create()
+	{
+		this.m.ID = "actives.rf_gouge";
+		this.m.Name = "Gouge";
+		this.m.Description = "A well-placed gouging attack that is likely to inflict debilitating injuries.";
+		this.m.KilledString = "Cut down";
+		this.m.Icon = "skills/rf_gouge_skill.png";
+		this.m.IconDisabled = "skills/rf_gouge_skill_sw.png";
+		this.m.Overlay = "rf_gouge_skill";
+		this.m.SoundOnUse = [
+			"sounds/combat/strike_01.wav",
+			"sounds/combat/strike_02.wav",
+			"sounds/combat/strike_03.wav"
+		];
+		this.m.SoundOnHit = [
+			"sounds/combat/strike_hit_01.wav",
+			"sounds/combat/strike_hit_02.wav",
+			"sounds/combat/strike_hit_03.wav"
+		];
+		this.m.Type = ::Const.SkillType.Active;
+		this.m.Order = ::Const.SkillOrder.OffensiveTargeted;
+		this.m.IsSerialized = false;
+		this.m.IsActive = true;
+		this.m.IsTargeted = true;
+		this.m.IsStacking = false;
+		this.m.IsAttack = true;
+		this.m.IsIgnoredAsAOO = true;
+		this.m.IsTooCloseShown = true;
+		this.m.IsWeaponSkill = true;
+		this.m.InjuriesOnBody = ::Const.Injury.CuttingBody;
+		this.m.InjuriesOnHead = ::Const.Injury.CuttingHead;
+		this.m.DirectDamageMult = 0.3;
+		this.m.ActionPointCost = 6;
+		this.m.FatigueCost = 30;
+		this.m.MinRange = 1;
+		this.m.MaxRange = 2;
+		this.m.ChanceDecapitate = 75;
+		this.m.ChanceDisembowel = 50;
+		this.m.ChanceSmash = 0;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.getDefaultTooltip();
+		ret.push({
+			id = 6,
+			type = "text",
+			icon = "ui/icons/hitchance.png",
+			text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorRed(this.getContainer().getActor().getCurrentProperties().IsSpecializedInCleavers ? "50%" : "33%") + " lower [threshold|Concept.InjuryThreshold] to inflict [injuries|Concept.InjuryTemporary]")
```

Comment:
comma at the end

Reply:
I don't think a comma here necessarily makes any sense. I can understand commas for tables/arrays where an expansion is later expected but for tooltips this is a fixed setup. So does it really make any sense to add a comma here?

Reply:
it's a great candidat for a soft rule: `in tables with a set amount of elements and order for them, the last declared element must not need a comma because it is not expected that another element is ever inserted after it.`

However we can't predict whether `text` will really always be the last element. Maybe we in msu/reforged improve the tooltip framework with more optional values.